src/applications/vaos/components/layout/ClaimExamLayout.jsx

```diff
@@ -69,25 +72,44 @@ export default function ClaimExamLayout({ data: appointment }) {
             </div>
           )}
       </When>
-      <What>{typeOfCareName || 'Type of care information not available'}</What>
+      <What>{typeOfCareName}</What>
       <Where
         heading={
           APPOINTMENT_STATUS.booked === status && !isPastAppointment
             ? 'Where to attend'
             : 'Where'
         }
       >
-        {!!facility === false && (
-          <>
-            <span>Facility details not available</span>
-            <br />
-            <NewTabAnchor href="/find-locations">
-              Find facility information
-            </NewTabAnchor>
-            <br />
-            <br />
-          </>
-        )}
+        {/* When the services return a null value for the facility (no facility ID) for all appointment types */}
```

Comment:
The facility logic in `ClaimExamLayout.jsx`, `VideoLayoutVA.jsx` and `InPersonLayout.jsx` should probably be pulled out and placed in a component on its own. Could simplify unit tests as well as future code changes. Didn't want to introduce that refactor this late in the review process so will open another issue for that.